src/test/ui/lint/used_parens_remove_json_suggestion.rs

```diff
@@ -0,0 +1,60 @@
+// compile-flags: --error-format pretty-json -Zunstable-options
```

Comment:
Not sure why you're using `pretty-json` here instead of `#![deny(unused_parens)]` + `//~ ERROR` annotations (see https://rust-lang.github.io/rustc-guide/tests/adding.html)

Reply:
Sorry, I am referencing the old `test/lint/unused_parens_json_suggestion.rs` test file.

Reply:
Maybe we can update style of both file later?

Reply:
Can you file an issue for updating the style?

Reply:
OK, new issue opened, [issue-63237](https://github.com/rust-lang/rust/issues/63237).